### PR TITLE
italian translation

### PR DESCRIPTION
**Pull request by napo**

### CODE REVIEW

File: superset/translations/it/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -2,1800 +2,803 @@
...
+#: superset/viz.py:1394
 msgid "Heatmap"
-msgstr ""
+msgstr "Mappa di Intensità"
 
-#: superset/viz.py:1410
+#: superset/viz.py:1445
 msgid "Horizon Charts"
-msgstr ""
+msgstr "Grafici Orizzontali"
```
Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: Are this really `Orizzontali` and not `dell'orizzonte` if we are going to translate it literally?

**Review by xrmx — COMMENTED** [review]
Thanks for doing the work!
Could you please add an entry for italian here too? https://github.com/airbnb/superset/blob/master/superset/config.py#L145

**Comment by xrmx** [comment]
BTW if you are using superset in FBK it would be really cool to have you added to https://github.com/airbnb/superset/blob/master/INTHEWILD.md

File: superset/translations/it/LC_MESSAGES/messages.po (hunk 1 above)
Comment (napo) [inline superset/translations/it/LC_MESSAGES/messages.po]: Thanks for you suggestion.
You are right: it's better use "Grafici d'orizzonte" or "Grafici horizon" or leave "Horizon Chart".
I have a question for you:
what do you think if we translate "Dashboard" with "Cruscotto"?
I like more use "dashboard" but the italian translation is "cruscotto".
My problem is for the translation of 'slice'. In italian language sounds like 'fetta' 
I like more use "widget" but this is english :)
What do you think?

Comment (napo) [inline superset/translations/it/LC_MESSAGES/messages.po]: Update of the italian translation with the suggestions made by @xrmx for the pull request  #2899

Comment (xrmx) [inline superset/translations/it/LC_MESSAGES/messages.po]: I think that it is fine for technical terms commonly used in italian to stay in english. Dashboard, slice and widget should stay all untranslated in my opinion.

**Pull request closed without merging**